src/styles/blocks/header.css

```diff
@@ -0,0 +1,55 @@
+.header {
+  display: flex;
+  align-items: center;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: white;
+  width: 100%;
+  height: 60px;
+  padding: 0 50px;
+}
+
+.header__logo {
+  margin: 10px;
+  padding-right: auto;
+  height: 40px;
+  width: 40px;
+}
+
+.header__nav {
+  display: flex;
+  flex-direction: row;
+  padding-right: 50px;
+}
+
+.header__nav__link {
+  color: #000;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 14.06px;
+  height: 15px;
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  margin-right: 20px;
+  padding: 22.5px 0;
+}
+
+.header__nav__link--is-active {
+  color: var(--link);
+}
+
+.header__nav__link--is-active::after {
+  content: '';
+  display: block;
+  height: 4px;
+  background-color: var(--link);
+  width: 100%;
+  margin-top: 18.5px;
+  border-radius: 8px;
+}
+
+.header__nav .header__nav__link:nth-child(4) {
```

Comment:
Remove this styling. The 4th link is a reference for how a link should look when hovered, not that it should be fixed in this styling.